overlord/snapstate/backend/copydata.go

```diff
@@ -20,18 +20,25 @@
 package backend
 
 import (
+	"errors"
+	"fmt"
+	"io/ioutil"
 	"os"
 
 	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/logger"
+	"github.com/snapcore/snapd/osutil"
+	snapshotbackend "github.com/snapcore/snapd/overlord/snapshotstate/backend"
```

Comment:
this kind of cross manager backend import though it doesn't cause circular imports is a bit confusing. I think the cleanest thing is to move AllUsers and its tests to the snap package